openwisp_controller/connection/base/models.py

```diff
@@ -44,6 +44,12 @@ def clean(self):
         self._validate_connector_schema()
 
     def _get_connector(self):
+        connectors = dict(app_settings.CONNECTORS)
+        try:
+            if connectors[self.credentials.connector] != 'SSH':
```

Comment:
```suggestion
            if connectors[self.credentials.connector] == 'SNMP':
```

Just for sake of improving readability.